### DISCUSSION TITLE
Customizing the IntelliSense extension

### DISCUSSION THREAD

**Discussion by o-t-w** [tailwindlabs/tailwindcss#9530] [Ideas]
I find the extension really useful but there are so many classes in Tailwind and there are many that I know I will never use. In the Tailwind version 2 docs it says you can disable the background opacity classes:
```
module.exports = {
  corePlugins: {
    backgroundOpacity: false,
  },
  ```
This seems to still work but is not mentioned in the v3 docs...
Is it possible to do something similar to other parts of Tailwind? Gradient `bg` classes for example? I would find the extension much more useful if it didn't recommend these classes. At the moment it suggests too much and I have to scroll through many classes to find what I want.

**Comment by wongjn** [comment]
The core plugin disabling feature still indeed exists in Tailwind v3 and the relevant documentation can be found on the [Configuration page](https://tailwindcss.com/docs/configuration#core-plugins).
